index.html

```diff
@@ -13,53 +13,186 @@
+                personalized reads; 
+                no accounts or sign-ups required
+            </p>
+            <div class="leng">
+                <ul>
+                    <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                    <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+                </ul>
+            </div>
+        </section>
+    </section>
 
-    <section class="main">
-        <p>
-            Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
+    <section class="all3">
+        <section>
+            <img class="two" src="images/SnapshootPortfolio77.jpg" alt="">
+            <h2 class="multipost">Tonic
+            </h2>
+            <div class="list2">
+                <ul class="inline2">
+                    <li><img class="words3" src="images/Frame2.jpg" alt=""></li>
+                </ul>
+            </div>
+        </section> 
+        <section>   
+            <p class="subtitle1">A daily selection is privaly 
+                personalized reads; 
+                no accounts or sign-ups required
+            </p>
+            <div class="leng">
+                <ul>
+                    <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                    <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+                </ul>
+            </div>
+        </section>    
+    </section>
+
+    <section class="all4">
+        <section>
+            <img class="two" src="images/SnapshootPortfolio)1.jpg" alt="">
+            <h2 class="multipost"> Multi - Post
+                Stories
+            </h2>
+            <div class="list2">
+                <ul class="inline2">
+                    <li><img class="words2" src="images/Frame2.jpg" alt=""></li>
+                </ul>
+            </div>
+        </section>
+        <section>
+            <p class="subtitle1">A daily selection is privaly 
+                personalized reads; 
+                no accounts or sign-ups required
+            </p>
+            <div class="leng">
+                <ul>
+                    <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                    <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+                </ul>
+            </div>
+        </section>    
     </section>
-    <br>
-    <footer class="contac">
-        <h5>Lets Connect</h5>
 
-    </footer>
+    <section class="all5">
+        <section>
+            <h2 class="about"> About Myself </h2>
+            <img class="sub1" src="images/PrimaryText.jpg" alt="">
+            <h3 class="lets">LETS CONNECT</h3>
+        </section>
```

Comment:
Kindly, the semantic SECTION tags are used to group each topic of your page (for example, home, works, about, etc), but using SECTION tags inside another SECTION is not a good practice, I recommend changing it to a DIV .